guides/v2.3/magento-functional-testing-framework/release-2/commands/mftf.md

```diff
@@ -0,0 +1,430 @@
+
+```bash
+mftf run:group example
+```
+
+#### Run and generate all tests
+
+The command runs all the generated tests.
+
+```bash
+mftf run:tests
+```
+
+## Reference
+
+### general
+
+#### `reset`
+
+##### Description:
+Clean any configuration files from the environment (not including .env), as well as any generated artifacts.  
+
+##### Usage:
+`reset [options]`
+
+##### Options:
+opt                | description|
+---|---|
+`--hard` | parameter to force reset of configuration files.
+
+  
+### build
+ 
+#### `build:project`     
+
+##### Description:
+Clone the example configuration files and build the Codeception project.
+
+### generate
+
+#### `generate:tests`
+
+##### Description:
+Generate PHP code from the tests defined in XML files. (Path is determined by constant `TESTS_MODULE_PATH`)
+
+##### Usage:
+`generate:tests [options] [--] [<name>]...`
+
+##### Arguments:
+*arg*                | description|
+|---|---|
+`name` | name(s) of specific tests to generate
+
+##### Options:
+opt                | description|
+|---|---|
+`--config=CONFIG`   | default, singleRun, or parallel [default: "default"]  
+`--force`           | force generation of tests regardless of Magento Instance Configuration  
+`--lines=LINES`     | Used in combination with a parallel configuration, determines desired group size [default: 500]  
+`--tests=TESTS`     | A parameter accepting a JSON string used to determine the test configuration  
+
+##### Examples of the JSON configuration
+
+The configuration to generate a single test with no suites:
+```json
+{  
+   "tests":[
+      "general_test1"     //Generate the "general_test1" test.
+      ],
+   "suites": null
+}
+```
+
+The configuration to generate a single test contained in the suite:
+```json
+{  
+   "tests": null,         // No tests outside the suite configuration will be generated.
+   "suites":{  
+      "sample":[          // The suite that contains the test.
+         "suite_test1"    // The test to be generated.
+      ]
+   }
+}
+```
+
+Complex configuration to generate few non-suite tests, a single test in a suite, and an entire suite:
+```json
+{  
+   "tests":[  
+      "general_test1",   
+      "general_test2",
+      "general_test3"
+   ],
+   "suites":{             //Go to suites.
+      "sample":[          //Go to the "sample" suite.
+         "suite_test1"    //Generate the "suite_test1" test.
+      ],
+      "sample2":[]        //Generate all tests in the "sample2" suite.
+   }
+}
+```
+
+The command that encodes the above configuration:
+
+```bash
+vendor/bin/robo generate:tests --tests "{\r\n\"tests\":[\r\n\"general_test1\",\r\n\"general_test2\",\r\n\"general_test3\"\r\n],\r\n\"suites\":{\r\n\"sample\":[\r\n\"suite_test1\"\r\n],\r\n\"sample2\":null\r\n}\r\n}"
+```
+
+{% include note.html
+type="info"
+content="The strings must be escaped and surrounded in quotes."
+%}
+
+#### `generate:suite`
+
+##### Description:
+Generates a single suite based on declaration in xml
```

Comment:
There is a collision.
The __Description__ tells about the _single_ suite, but the __Usage__ uses _multiple_ suites.

Reply:
Ahh good point, this actually allows for multiple suites to be generated depending on how many args are passed in, I will fix it.